### ISSUE TITLE
Preivew Pages from another assembly

### ISSUE THREAD

**Issue by No1e** [UXDivers/Gorilla-Player-Support#193]
Hi,

Lets say that I have core framework (PCL, Droid, iOS, UWP) that I share among my apps, and in this framework I have some pages.

I have reference to respective assemblies in my app and I would like to preview those pages from core framework, since they are using styles defined in app.

Is it possible to preview pages from another assembly? 

Thanks

**Comment by LeoHere** [comment]
No, currently this is not possible. Is there any reason why you can't preview it from the source code of your core framework?
